src/agents/realtime/model_events.py

```diff
@@ -84,6 +84,7 @@ class RealtimeModelInputAudioTranscriptionCompletedEvent:
 
     type: Literal["input_audio_transcription_completed"] = "input_audio_transcription_completed"
 
+
```

Comment:
can you remove these unrelated changes to make the review easier?

Reply:
> can you remove these unrelated changes to make the review easier?

Of course! I removed the final commit that had all the `make lint/check-format` changes that weren't part of the bug fix.